base/deprecated.jl

```diff
@@ -440,3 +440,11 @@ export float32_isvalid, float64_isvalid
 @deprecate (&)(x::Char, y::Char)  Char(UInt32(x) & UInt32(y))
 @deprecate (|)(x::Char, y::Char)  Char(UInt32(x) | UInt32(y))
 @deprecate ($)(x::Char, y::Char)  Char(UInt32(x) $ UInt32(y))
+
+# 11241
+
+@deprecate is_valid_char(ch::Union(Unsigned, Integer, Char)) isvalid(Char, ch)
+@deprecate is_valid_ascii(str::Union(Vector{UInt8}, ByteString)) isvalid(ASCIIString, str)
```

Comment:
Actually, that doesn't work.  All of those have a `Union` type, where one of the values requires the 2 argument form of `isvalid` to correctly disambiguate... for example, you can't tell what you want to validate a `ByteString` or a `Vector{UInt8}` for, it could be either `ASCIIString` or `UTF8String`.
I want to make sure, that even if somebody defines another method, such as `isvalid(Integer)`, that it is not confused with `isvalid(Char, Integer)`.